src/composables/use-ec-countdown/use-ec-countdown.ts

```diff
@@ -0,0 +1,71 @@
+import { createEventHook } from '@vueuse/core';
+import {
+  type MaybeRef, ref, unref, watch,
+} from 'vue';
+
+export default function useEcCountdown(secondsToGo: MaybeRef<number>) {
+  const secondsLeft = ref<number>(unref(secondsToGo));
```

Comment:
```suggestion
  const secondsLeft = ref(unref(secondsToGo));
```
type of secondsLeft will be infer from unref(secondsToGo) return type